src/dialogs/account-application/AdviserRequestDialog.vue

```diff
@@ -124,4 +128,24 @@ function onRequest() {
     },
   });
 }
+
+function focusNext(e: KeyboardEvent) {
+  const target = e.target as HTMLInputElement | null;
+
+  if (target) {
+    const inputs = Array.from(
+      target.form?.querySelectorAll('input') || []
+    ) as HTMLInputElement[];
+    console.log(inputs);
+    const index = inputs.indexOf(target);
+
+    if (index < inputs.length - 1) {
+      inputs[index + 1].focus();
+    } else if (isFormValid.value) {
+      onRequest();
+    }
+
+    e.preventDefault();
+  }
+}
```

Comment:
this should not be necessary because this already handled by q-form by using @submit event and q-btn type=submit, I will push the change so you can review and test in your end later

Reply:
Sir, I believe there may be a misunderstanding about how this code works If you're just referring to triggering the workflows, then you're correct, but this function also enables users to navigate through input fields using the ‘Enter’ key. This eliminates the need to manually click each field, streamlining the user experience.

Reply:
yes, understand the UX you trying to achieve, which q-form already is handling either `{TAB}` or `{ENTER}` keys, with validation of the rules. Although the down sides is, it skips the input when the value on the input is already valid.

Reply:
it didn't work in my case that's why I hard-coded it but I can try it again.